coalib/core/Core.py

```diff
@@ -0,0 +1,175 @@
+import asyncio
+import concurrent.futures
+import functools
+import logging
+import multiprocessing
+
+
+def get_cpu_count():
+    try:
+        return multiprocessing.cpu_count()
+    except NotImplementedError:  # pragma: no cover
+        # cpu_count is not implemented for some CPU architectures/OSes
+        return 1
+
+
+def cleanup_bear(bear,
+                 result_callback,
+                 running_tasks,
+                 event_loop,
+                 executor):
```

Comment:
why so many params? are you trying to have the same signature across all functions?